File: Framework/Qt/Python/Chapter_01/QT01_03.py
"""
Setup Qt application software with QLabel widget as a main window.
"""

from PySide2.QtWidgets import QApplication, QLabel

if __name__ == '__main__':
    app = QApplication()

    # DECLARE label widget with "Hello World!".
    label = QLabel("Hello World!")

    '''
    IMPORTANT: Widget without any parent widget is a top - level widget, called window.
	'''	

    # PRESENT label widget and its child widgets of application software.
    label.show()

    app.exec_()

File: Framework/Qt/Cpp/Chapter_01/QT01_03/QT01_03.cpp
/*
Setup Qt application software with QLabel widget as a main window.
*/

#include <qapplication.h>
#include <qlabel.h>

int main(int argc, char **argv) {
	QApplication app(argc, argv);

	// DECLARE label widget with "Hello World!".
	QLabel label("Hello World!");

	/*
	IMPORTANT: Widget without any parent widget is a top - level widget, called window.
	*/

	// PRESENT label widget and its child widgets of application software.
	label.show();

	return app.exec();
}